lib/models/MailPiece.dart

```diff
@@ -0,0 +1,10 @@
+class MailPiece {
+  int _mail_ID;
+  DateTime _informedDate;
+  String _LogoText;
+  AddressObject _senderAddress;
```

Comment:
I'd make this a `String`. Otherwise, you'll need to import whatever library provides this type.